### DISCUSSION TITLE
目录、引用字体颜色

### DISCUSSION THREAD

**Discussion by LoganLiu66** [mengchaoheng/SCUT_thesis#40] [Show and tell]
为了让目录、引用变成黑色字体可以参考如下设置：
找到使用hyperref的地方，然后设置更改`colorlinks=false`

**Comment by mengchaoheng** [comment]
这个在文档里有说明，有什么问题吗？

**Reply by sikouhjw** [reply]
猜测：单纯的不看文档，自己摸索出答案，想分享一下

**Reply by mengchaoheng** [reply]
回复错了，sorry

**Reply by sikouhjw** [reply]
> 不看文档…，额，好吧。那你去讨论区发。这里是issues

事实上，这里就是讨论区（
